js/apps/admin-ui/cypress/e2e/identity_providers_test.spec.ts

```diff
@@ -88,11 +88,6 @@ describe("Identity provider test", () => {
           alias: "linkedin-openid-connect",
         },
         { testName: "Microsoft", displayName: "Microsoft", alias: "microsoft" },
-        {
-          testName: "Openshift-v3",
-          displayName: "Openshift v3",
-          alias: "openshift-v3",
-        },
         {
           testName: "Openshift-v4",
           displayName: "Openshift v4",
```

Comment:
Perhaps we also re-name labels with 'Openshift v4' to only 'Openshift'?

Reply:
Probably wouldn't do that since it would be a potential breaking change, and I'd say it doesn't hurt to just leave it as is.